config/printer-geeetech-A10T-A20T-2021.cfg

```diff
@@ -0,0 +1,272 @@
+
+[safe_z_home]
+home_xy_position: 100, 100  # Change coordinates to the center of your print bed
+speed: 50
+z_hop: 10  # Move up 10mm
+z_hop_speed: 5
+
+[stepper_x]
+enable_pin: !PC2
+dir_pin: !PG2
+step_pin: PC0
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA2 # there are external pull ups
+position_endstop: 0
+position_max: 220 # for A10/M/T / change to 250 for A20/M/T
+homing_speed: 40
+
+[stepper_y]
+enable_pin: !PA7
+dir_pin: !PC4
+step_pin: PC6
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA6 # there are external pull ups
+position_endstop: 0
+position_max: 220 # for A10/M/T / change to 250 for A20/M/T
+homing_speed: 40
+
+[stepper_z]
+enable_pin: !PA5
+dir_pin: PA1
+step_pin: PA3
+microsteps: 16
+rotation_distance: 8
+#endstop_pin: probe:z_virtual_endstop # uncomment for BLTouch/3DTouch
+endstop_pin: !PC7 # comment for BLTouch/3DTouch
+position_endstop: 0 # comment for BLTouch/3DTouch
+position_max: 230 # for A10/M/T / change to 250 for A20/M/T
+position_min: -5
+homing_speed: 20
+
+[extruder]
+enable_pin: !PB6
+dir_pin: PL5
+step_pin: PL3
+microsteps: 16
+rotation_distance: 8 # Needs to be optimized: https://www.klipper3d.org/Rotation_Distance.html#calibrating-rotation_distance-on-extruders
+nozzle_diameter: 0.4
+filament_diameter: 1.750
+heater_pin: PB4
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PK3
+min_temp: 0
+max_temp: 250
+max_extrude_only_distance: 200.0
+# Parameters for stock hotend on A10M
+# Please recalibrate according to https://www.klipper3d.org/Config_checks.html#calibrate-pid-settings
+control = pid
+pid_kp = 54.722
+pid_ki = 4.800
+pid_kd = 155.958
+
+[extruder_stepper extruder_1]
+extruder:
+enable_pin: !PL1
+dir_pin: PL2
+step_pin: PL0
+microsteps: 16
+rotation_distance: 8 # Needs to be optimized: https://www.klipper3d.org/Rotation_Distance.html#calibrating-rotation_distance-on-extruders
+
+[extruder_stepper extruder_2]
+extruder:
+enable_pin: !PG0
+dir_pin: PL4
+step_pin: PL6
+microsteps: 16
+rotation_distance: 8 # Needs to be optimized: https://www.klipper3d.org/Rotation_Distance.html#calibrating-rotation_distance-on-extruders
+
+[heater_bed]
+heater_pin: PG5
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PK2
+min_temp: 0
+max_temp: 120
+# Parameters for `SuperPlate` on A10M
+# Please recalibrate according to https://www.klipper3d.org/Config_checks.html#calibrate-pid-settings
+control = pid
```

Comment:
same as extruder, please convert these to : instead of = so as not to confuse,

Thanks
James